src/benchmark.cc

```diff
@@ -315,6 +343,16 @@ bool IsZero(double n) {
   return std::abs(n) < std::numeric_limits<double>::epsilon();
 }
 
+void MergeResults(State& st, ThreadTimer* timer, ThreadManager* manager)
+{
+  ThreadManager::Result& results = manager->results;
+  results.cpu_time_used += timer->cpu_time_used();
+  results.real_time_used = std::max(results.real_time_used, timer->real_time_used());
```

Comment:
Note to myself: leftover of another fix. 
Here it still must be `results.real_time_used += timer->real_time_used();`